packages-exp/auth-exp/src/core/strategies/custom_token.ts

```diff
@@ -56,3 +60,30 @@ export async function signInWithCustomToken(
   await authInternal._updateCurrentUser(cred.user);
   return cred;
 }
+
+/**
+ * Sets a custom token provider, which is invoked when a new Firebase ID token is requested and no
+ * refresh token is present (i.e., in passthrough mode). This callback should be implemented to
```

Comment:
Break the new sentence into a new paragraph, separated from the top line by `@remarks`